CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ A *config* knows about the code style rules and the files and directories that m
 * Make changes. Please remember that **all** changes have to be covered by tests.
    * if you work on a bug fix, please start with reproducing the problem by adding failing test case(s). When you have failing test case(s), you can [create pull request](#opening-a-pull-request) just to reproduce fail in the CI. Then you can provide fix _in the subsequent commits_, it will make code review easier. It's allowed to modify existing test cases in bug fix pull request, but *only if* current behavior is proved to be invalid.
    * if you work on existing fixers then don't change existing test cases, because these are contract between the maintainers and users (they ensure how tool works). Add new test cases that cover provided changes - preferred way of defining test cases is with [data provider](https://docs.phpunit.de/en/10.0/writing-tests-for-phpunit.html#data-providers) which uses `yield` with proper case description as a key (e.g. `yield 'Some specific scenario' => ['some', 'example', 'data'];`). Codebase may still contain test cases in different format, and it's totally acceptable to use `yield` approach next to existing `return` usages.
-* Update documentation: `composer docs`. This requires the highest version of PHP supported by PHP CS Fixer. If it is not installed on your system, you can run it in a Docker container instead: `docker compose run php-8.2 php dev-tools/doc.php`.
+* Update documentation: `composer docs`. This requires the highest version of PHP supported by PHP CS Fixer. If it is not installed on your system, you can run it in a Docker container instead: `docker-compose run php-8.2 php dev-tools/doc.php`.
```

Comment:
I'm not about this one, I saw @Wirone did the opposite: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7450/commits/88f703fd7f750b327f3929c91ec4fed4f53d6b8a

Reply:
:+1:  lets see what Wirone thinks,
I get:
```
$ docker compose
docker: 'compose' is not a docker command.
```